.github/workflows/build-and-test-arm.yaml

```diff
@@ -6,7 +6,15 @@ on:
   workflow_dispatch:
 
 jobs:
+  pre_condition:
```

Comment:
Or I think this is good.

```suggestion
  prevent-running-on-forks:
```

Reply:
Sorry, I overlooked this comment. 1f1451f6